Symfony/CS/Fixer/PSR2/MethodArgumentSpaceFixer.php

```diff
@@ -53,12 +53,20 @@ public function fix(\SplFileInfo $file, $content)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function getDescription()
+    {
+        return 'In method arguments and method call, there MUST NOT be a space before each comma and there MUST be one space after each comma.';
+    }
+
+    /**
      * Method to insert space after comma and remove space before comma.
      *
-     * @param Tokens $tokens tokens to handle
-     * @param int    $index  index of token
+     * @param Tokens $tokens
+     * @param int    $index
      */
-    public function fixSpace(Tokens $tokens, $index)
+    private function fixSpace(Tokens $tokens, $index)
```

Comment:
Method is public at 1.4 and we MUST NOT change it visibility at 1.x line.
Please just reorder them, visibility changes could be applied into 2.0-dev